charts/platform/values.yaml

```diff
@@ -199,6 +199,8 @@ server:
           privateKeyPath: /etc/opentdf/kas/kas-ec-private.pem
           publicKeyPath: /etc/opentdf/kas/kas-ec-cert.pem
 services:
+  entityresolution:
+    enabled: false
```

Comment:
@imdominicreed Can we stub out the required configuration here.

Reply:
Yep, probably a good idea. Let me know if you have  any problems the stubbed values.